main.cpp

```diff
@@ -115,16 +98,29 @@ Q_DECL_EXPORT int main(int argc, char *argv[])
 
     auto view = createView("Main.qml");
 
-    QQmlContext *context = view->rootContext();
-    context->setContextProperty(QStringLiteral("coverWindowAccessor"), new CoverWindowFetcher(view->engine()));
-    context->setContextProperty(QStringLiteral("applicationWindow"), view.data());
-
     //% "Documents"
     Q_UNUSED(QT_TRID_NOOP("sailfish-office-ap-name"))
 
+    bool preStart = false;
+    QString fileName;
+
+    for (int i = 1; i < argc; ++i) {
+        if (QString(argv[i]) == "-prestart") {
```

Comment:
Use ```QStringLiteral("-prestart")``` in the comparison (http://doc.qt.io/qt-5/qstring.html#QStringLiteral).

I'd prefer using app->arguments() QStringList but this does the job as well.